changelogs/unreleased/3832-make-mypy-diff-error-double-bracket-not-found.yml

```
@@ -0,0 +1,6 @@
+description: Explicitly use bash instead of assuming sh symlinking to it to
+    prevent errors for shells where the [[ keyword is not supported
+change-type: patch
+destination-branches: [master, iso5]
```

I think this can safely go into iso4 as well.
```suggestion
destination-branches: [master, iso5, iso4]
```